sim/battle.ts

```diff
@@ -3017,7 +3017,7 @@ export class Battle {
 		return team as PokemonSet[];
 	}
 
-	showOpenTeamSheets(hideFromSpectators = false) {
+	showOpenTeamSheets() {
```

Comment:
... I guess these hunks can always be reverted if you need to change the setting in future for any reason...

Reply:
Since this is for VGC only, I'm not terribly worried about dropping it. Improves performance by 0.0001 seconds or whatever by not having to check the if statement.